supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -430,7 +430,7 @@ These attributes identify users and groups as separate entities.
 [discrete]
 [[producer]]
 ==== image:images/yes.png[yes] producer (noun)
-*Description*: 1) In Red Hat AMQ, a producer is a client that sends messages. 2) In Red Hat Fuse, a producer is an endpoint that acts as the source of messages exiting a Camel route. It can create and send processed messages to their target destination, such as external systems or services. The producer populates the messages it creates with data that is compatible with the target destination. A route can have multiple producers.
+*Description*: 1) In Red Hat AMQ, a _producer_ is a client that sends messages. 2) In Red Hat Fuse, a producer is an endpoint that acts as the source of messages exiting a Camel route. It can create and send processed messages to their target destination, such as external systems or services. The producer populates the messages it creates with data that is compatible with the target destination. A route can have multiple producers.
```

Comment:
```suggestion
*Description*: (1) In Red Hat AMQ, a _producer_ is a client that sends messages. (2) In Red Hat Fuse, a producer is an endpoint that acts as the source of messages exiting a Camel route. It can create and send processed messages to their target destination, such as external systems or services. The producer populates the messages it creates with data that is compatible with the target destination. A route can have multiple producers.
```